src/libsync/configfile.cpp

```diff
@@ -101,6 +101,7 @@ static const char useNewBigFolderSizeLimitC[] = "useNewBigFolderSizeLimit";
 static const char confirmExternalStorageC[] = "confirmExternalStorage";
 static const char moveToTrashC[] = "moveToTrash";
 
+static const char allowChecksumValidationFailC[] = "allowChecksumValidationFail";
```

Comment:
@mgallien I don't know. Other parameters are in `OCC`, so, I thought I'd follow a pattern and put `allowChecksumValidationFailC` into `OCC `as well. Any reason for it to be in a different namespace?

Reply:
my point is that sometime we are using `static const char` that could be `static constexpr char` I guess and sometime we are using an anonymous namespace
this is making me feel lost between two different styles
it would be pretty nice that at least new code is trying to be consistent and use always anonymous namespace for example

Reply:
@mgallien I will factor it out into an anonymous namespace.